docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/07_ReactJS_Redux_and_GraphQL.md

```diff
@@ -15,6 +15,12 @@ There are some several members of this ecosystem that all work together to provi
 
 All of these pillars of the frontend application can be customised, giving you more control over how the admin interface looks, feels, and behaves.
 
+<div class="alert" markdown="1">
+These technologies underpin the future of SilverStripe CMS development, but their current implementation is
+_experimental_. Our APIs are not expected to change drastically between releases, but they are excluded from
+our Semver commitments for the time being. Any breaking changes will be clearly signalled in release notes.
```

Comment:
Maybe say “semantic versioning” with a link to semver.org

Reply:
Good call - added 👍